charts/merlin/templates/merlin-environments.yaml

```diff
@@ -8,12 +8,6 @@ metadata:
   labels:
     {{- include "merlin.labels" . | nindent 4 }}
 stringData:
-  environment.yaml: |-
-{{- if .Values.environmentConfigs }}
-{{ toYaml .Values.environmentConfigs | indent 4 }}
-{{- end }}
-  imageBuilderK8sConfig: |-
-{{- if .Values.imageBuilder.k8sConfig }}
-{{ .Values.imageBuilder.k8sConfig | indent 4 }}
-{{- end }}
```

Comment:
This block is no longer needed since the image builder cluster credentials are now stored together in the Merlin config secret.